backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/vdsbroker/CreateBrokerVDSCommand.java

```diff
@@ -73,14 +74,14 @@ private Map<String, Object> createInfo() {
             createInfo.put("memoryConfVolume", memoryConfPDIV);
         }
 
-        String tpmData = vmInfoBuildUtils.tpmData(vm.getId());
+        SecretValue<String> tpmData = vmInfoBuildUtils.tpmData(vm.getId());
         if (tpmData != null) {
-            createInfo.put(VdsProperties.tpmData, tpmData);
+            createInfo.put(VdsProperties.tpmData, tpmData.getValue());
```

Comment:
Also here, `tpmData.getValue()` may be `null`.

Reply:
Done.